api/pom.xml

```diff
@@ -2,7 +2,7 @@
    <parent>
       <groupId>org.openmrs</groupId>
       <artifactId>openmrs</artifactId>
-      <version>1.12.0.SPRING-SNAPSHOT</version>
+      <version>1.12.0-SPRING4-SNAPSHOT</version>
```

Comment:
@kristopherschmidt, I don't envision a need to test 1.12.0-SNAPSHOT with spring 3 anymore. If needed one can do that with extra care by checking out a commit before spring 4 and doing mvn clean install on the full project (not only the web) so that the core-api jar will be on spring 3 as well. You can also change version to 1.12.0-SPRING3-SNAPSHOT locally to be 100% sure. Also we will continue to have the 1.11.x branch which will remain spring 3.